tools/blobtoolkit/blobtoolkit.xml

```diff
@@ -189,6 +174,7 @@
             </when>
             <when value="add">
                 <param name="blobdir" type="data" format="tgz" label="Blobdir.tgz file" help="This file should be generated by Blobtool create" />
+                <param argument="--taxdump" type="data" format="tar,tgz,gz" label="NCBI taxdump directory" help="It should be compressed in tar.gz format"/>
```

Comment:
Do you think we should add a galaxy data type for it? Maybe even zip? Or should we stick to this for the time being?

Reply:
Indeed it seems to be better to use the zip file, because the tar.gz is recognized by Galaxy as gzip, triggering in the tool.

Reply:
Unfortunately the container doesn't include zip and unzip...